redis/src/aio/mod.rs

```diff
@@ -140,6 +140,20 @@ where
         }
     }
 
+    if connection_info.client_name.is_some() {
```

Comment:
this implementation sends the CLIENT SETNAME command and only after it completes the CLIENT SETINFO command is sent. Wouldn't it be better to send both concurrently?

Reply:
You mean add it to client_set_info_pipeline?  yes, it can be done. Why dont we have pipelined DB SELECT then?

Reply:
will move to client_set_info_pipeline

Reply:
beyond scope of this PR